chart/epinio/templates/default-app-chart.yaml

```
@@ -12,4 +12,7 @@ metadata:
 spec:
   shortDescription: Epinio standard deployment
   description: Epinio standard support chart for application deployment
-  helmChart: https://github.com/epinio/helm-charts/releases/download/epinio-application-0.1.24/epinio-application-0.1.24.tgz
+  helmChart: https://github.com/epinio/helm-charts/releases/download/epinio-application-0.1.25/epinio-application-0.1.25.tgz
```

```suggestion
  helmChart: https://github.com/epinio/helm-charts/releases/download/epinio-application-0.1.25/epinio-application-0.1.25.tgz
```
Is the version specified in this url the same one than the one specificied in the Chart.yaml? 
If yes, then it would be better to use templating.

Hardcoded version in template files are not easy to find

---

I have restored the old code, as @enrichman said, this specific update of pointed version should be handled automatically by an automatic PR.

---

Ha ok sorry

---

@giubacc I was not probably clear that we need to wait the update also to add the settings part, or the test will fail.
Basically we cannot touch the Epinio chart until we have the new release ready.